src/common/DownloadManager.hpp

```diff
@@ -21,7 +21,7 @@ class DownloadManager : public QObject
 
 private:
     QNetworkAccessManager *manager;
-    QNetworkReply *reply;
+    QNetworkReply *downloadReply;
```

Comment:
does this fix a warning?

Reply:
and if so, which one

Reply:
As it says in the commit message:

`warning C4458: declaration of 'reply' hides class member`

This is on the Windows build.